app/scripts/modules/amazon/src/loadBalancer/configure/choice/awsLoadBalancerChoice.modal.ts

```diff
@@ -0,0 +1,47 @@
+import { IComponentController, module } from 'angular';
+
+import { Application } from '@spinnaker/core';
+
+import { IAwsLoadBalancerConfig, LoadBalancerTypes } from './LoadBalancerTypes';
+
+import './awsLoadBalancerChoice.less'
+
+class AwsLoadBalancerChoiceCtrl implements IComponentController {
+  public choices: IAwsLoadBalancerConfig[];
+  public choice: IAwsLoadBalancerConfig;
+
+  constructor(public $uibModal: any,
+              public $uibModalInstance: any,
+              private application: Application) {
```

Comment:
I was going to blame it on copy/paste, but what i copied it from was all private. So... no reason?